x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/pattern/pattern_summary/stats_rollup/index.tsx

```diff
@@ -120,23 +120,25 @@ const StatsRollupComponent: React.FC<Props> = ({
         </IndicesStatContainer>
       </EuiFlexItem>
 
-      <EuiFlexItem grow={false}>
-        <IndicesStatContainer>
-          <EuiToolTip
-            content={
-              pattern != null
-                ? i18n.INDICES_SIZE_PATTERN_TOOL_TIP(pattern)
-                : i18n.TOTAL_SIZE_TOOL_TIP
-            }
-          >
-            <EuiStat
-              description={sizeDescription}
-              title={sizeInBytes != null ? formatBytes(sizeInBytes) : EMPTY_STAT}
-              titleSize={STAT_TITLE_SIZE}
-            />
-          </EuiToolTip>
-        </IndicesStatContainer>
-      </EuiFlexItem>
+      {Number.isInteger(sizeInBytes) && (
```

Comment:
sizeInBytes is `number | undefined`, and not all numbers are integers, so why not just do: `sizeInBytes != null`?

Reply:
👍  I saw `Number.isInteger` has been used in the dq code, so followed that pattern, but sizeInBytes != null is simplier.